### DISCUSSION TITLE
How to set a default layout in qtile as monadwide

### DISCUSSION THREAD

**Discussion by bhavukshm** [qtile/qtile#2501] [Q&A] [answered]
Whenever i start my qtile every window opens with layout max by default. I want to change it so that i will only get monadwide whenever i opens a new window. Is there a setting for that, if yes than where it is and what is its name

**Comment by ramnes** [comment, marked as answer]
Just change the layouts order in your configuration?

**Reply by bhavukshm** [reply]
Thanks for your help
